src/main/java/pl/knpj/servlet/model/Answer.java

```diff
@@ -0,0 +1,42 @@
+package pl.knpj.servlet.model;
+
+
+import java.util.Collection;
+
+/**
+ * Created by vadikms on 29.11.16.
+ */
+public class Answer {
```

Comment:
To jest jedno rozwiazanie (dla kazdego pojo oddzielna klasa)
A drugim moze byc jedna klasa ze wszystkimi i np z klasami wewnetrznymi dla konkretnych pojo lub po prostu zwyklymi stalymi ;)